dolphinscheduler-dist/release-docs/LICENSE

```diff
@@ -492,6 +492,8 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
     slf4j-api 1.7.5: https://mvnrepository.com/artifact/org.slf4j/slf4j-api/1.7.5, MIT
     animal-sniffer-annotations 1.14 https://mvnrepository.com/artifact/org.codehaus.mojo/animal-sniffer-annotations/1.14, MIT
     checker-compat-qual 2.0.0 https://mvnrepository.com/artifact/org.checkerframework/checker-compat-qual/2.0.0, MIT + GPLv2
+    moco-core 1.2.0: https://mvnrepository.com/artifact/com.github.dreamhead/moco-core/1.2.0, MIT
+    moco-runner 1.2.0: https://mvnrepository.com/artifact/com.github.dreamhead/moco-runner/1.2.0, MIT
```

Comment:
Why should we add these, they are `test` scope dependencies

Reply:
thanks for your remind, i will remote the moco 2 line registion